packages/svelte/scripts/globals-extractor.js

```diff
@@ -16,7 +16,7 @@ const GLOBAL_TS_PATH = './src/compiler/utils/globals.js';
 const SPECIALS = ['global', 'globalThis', 'InternalError', 'process', 'undefined'];
 
 const get_url = (name) =>
-	`https://raw.githubusercontent.com/microsoft/TypeScript/main/lib/lib.${name}.d.ts`;
+	`https://raw.githubusercontent.com/microsoft/TypeScript/main/src/lib/${name}.d.ts`;
```

Comment:
Thanks for the contribution!!

I noticed that both the old and new URL give 404 not found on visiting them in my browser. Maybe a mistake?

Reply:
I tried click the link in my article, but there's nothing wrong.
```
/// <reference lib="es2020" />
/// <reference lib="es2021.promise" />
/// <reference lib="es2021.string" />
/// <reference lib="es2021.weakref" />
/// <reference lib="es2021.intl" />
```

Reply:
Aha! Yeah my bad, lgtm